packages/arb-token-bridge-ui/src/hooks/useArbQueryParams.tsx

```diff
@@ -37,6 +42,8 @@ export const useArbQueryParams = () => {
     ]
   */
   return useQueryParams({
+    from: ChainParam,
+    to: ChainParam,
```

Comment:
```suggestion
    sourceChain: ChainParam,
    destinationChain: ChainParam,
```

Reply:
We should also allow just passing in a raw chain id (for example for custom Orbit chains that don't have a preset key)